instance/models/openedx_instance.py

```diff
@@ -274,9 +274,14 @@ def get_load_balancer_configuration(self):
         """
         if not self.active_appserver:
             return self.get_preliminary_page_config(self.ref.pk)
+        ip_address = self.active_appserver.server.public_ip
+        if not ip_address:
+            # The active appserver doesn't have a public IP address.  This means that the server
+            # has been terminated, so we don't show the preliminary page in this case, and simply
+            # deconfigure the backend instead.
+            return [], []
```

Comment:
@smarnach I don't think it makes sense to consider an appserver as active if it is deprovisioned? Either there is a newer appserver available and it should have been made active before the active appserver got deprovisioned, or the instance itself has been shutdown, and in this case we shouldn't be calling this method at all?